src/tests/Makefile.am

```diff
@@ -74,9 +74,18 @@ check-container-debian-sid-image: check-container-%-image:
 	echo "RUN apt-get install -y autoconf automake pkg-config intltool libglib2.0-dev \
```

Comment:
it's still not clear, why you cannot manually start dbus-borker and polkit in the docker-init-less container.

Apparently it already works, with NetworkManager (and dbus-broker)?

Anyway. Using systemd inside the container is probably a better choice anyway. So ok. Would just have been nice to understand what the problem is.

Reply:
> it's still not clear, why you cannot manually start dbus-borker and polkit in the docker-init-less container.

You could. Or we could be more like the actual distribution and just start systemd. Polkit also does not like to run without being dbus activated. Instead of hacking it up I decided to just run like the actual distro.